src/html/HTMLWriter.jl

```diff
@@ -502,6 +502,7 @@ struct HTML <: Documenter.Writer
     size_threshold_ignore::Vector{String}
     example_size_threshold::Int
     inventory_version::Union{String, Nothing}
+    search_size_threshold_warn::Int
```

Comment:
I'd group the fields together.

```suggestion
    search_size_threshold_warn::Int
    inventory_version::Union{String, Nothing}
```

Would also need a change to the `new()` call below.